AmplifyPlugins/DataStore/Tests/DataStoreHostApp/AWSDataStorePluginLazyLoadTests/LL1/Comment4V2+Schema.swift

```diff
@@ -32,7 +32,7 @@ extension Comment4V2 {
     model.fields(
       .field(comment4V2.id, is: .required, ofType: .string),
       .field(comment4V2.content, is: .required, ofType: .string),
-      .belongsTo(comment4V2.post, is: .optional, ofType: Post4V2.self, targetNames: ["postID"]),
+      .belongsTo(comment4V2.post, is: .optional, ofType: Post4V2.self, associatedWith: Post4V2.keys.id, targetNames: ["postID"]),
```

Comment:
let's revert this change as well, since codegen is not currently generating this with `associatedWith`